Gemfile

```diff
@@ -74,7 +74,8 @@ end
 gem 'solidus', '~> 4.2', '>= 4.2.3'
 
 
-gem "solidus_auth_devise", "~> 2.5"
+ gem "solidus_auth_devise", "~> 2.5"
```

Comment:
Why we have changes here?

Reply:
Probably an extra space. This was left over from the first attempt to solve the issue by disabling gem authorization. But then the login for admin stopped working. So I uncommented this gem, but the space was left)).